XenAdmin/Dialogs/LicenseManager/LicenseStatus.cs

```diff
@@ -213,9 +213,9 @@ internal static bool PoolIsMixedFreeAndExpiring(IXenObject xenObject)
                                    into g
                                    select new { ExpiryDate = g.Key, Hosts = g };
 
-                if(expiryGroups.Count() > 1)
+                if (expiryGroups.Count() > 1)
                 {
-                    expiryGroups.OrderBy(g => g.ExpiryDate);
+                    expiryGroups = expiryGroups.OrderBy(g => g.ExpiryDate).ToList();

```

Comment:
I just noticed this: `.ToList()` should be moved to line 214, i.e. `var expiryGroups = (...).ToList()` to avoid multiple enumerations. Then the if check can be `expiryGroups.Count`.

Reply:
Apart from this, I think we're finally done with this PR!